### DISCUSSION TITLE
[Support]: Understanding models

### DISCUSSION THREAD

**Discussion by mabed-fr** [blakeblackshear/frigate#9147] [Ask A Question]
### Describe the problem you are having

I don't really have a problem and I'm not really familiar with AI (in general)

I would like to understand.

What model are you using (cpu and edgetpu)? Are there any models that are part of the frigate project? Are they updated with the different frigate images?

Does contributing to frigate+ help learning these models?

Thank you for your answers

### Version

0.12.1-367d72

### Frigate config file

```yaml
No needed
```

### Relevant log output

```shell
No needed
```

### FFprobe output from your camera

```shell
No needed
```

### Frigate stats

```json
{"clim":{"camera_fps":10.1,"capture_pid":226,"detection_enabled":1,"detection_fps":0.0,"ffmpeg_pid":231,"pid":224,"process_fps":10.1,"skipped_fps":0.0},"cpu_usages":{"%Cpu(s):":{"cpu":"id,","mem":"0.8"},"1":{"cpu":"0.0","mem":"0.0"},"100":{"cpu":"0.7","mem":"1.6"},"101":{"cpu":"0.0","mem":"0.0"},"118":{"cpu":"0.0","mem":"0.1"},"135":{"cpu":"0.0","mem":"0.0"},"136":{"cpu":"0.0","mem":"0.1"},"1387":{"cpu":"0.0","mem":"0.0"},"1417":{"cpu":"0.0","mem":"0.0"},"16":{"cpu":"0.0","mem":"0.0"},"17":{"cpu":"0.0","mem":"0.0"},"213":{"cpu":"0.0","mem":"0.5"},"219":{"cpu":"0.3","mem":"0.1"},"220":{"cpu":"0.0","mem":"0.6"},"221":{"cpu":"0.3","mem":"0.6"},"224":{"cpu":"1.3","mem":"0.6"},"226":{"cpu":"5.0","mem":"0.6"},"229":{"cpu":"0.0","mem":"0.1"},"231":{"cpu":"11.0","mem":"0.5"},"25":{"cpu":"0.0","mem":"0.0"},"26":{"cpu":"0.0","mem":"0.0"},"27":{"cpu":"0.0","mem":"0.0"},"28":{"cpu":"0.0","mem":"0.0"},"29":{"cpu":"0.0","mem":"0.0"},"30":{"cpu":"0.0","mem":"0.0"},"31":{"cpu":"0.0","mem":"0.0"},"32":{"cpu":"0.0","mem":"0.0"},"33":{"cpu":"0.0","mem":"0.0"},"42":{"cpu":"0.0","mem":"0.0"},"43":{"cpu":"0.0","mem":"0.0"},"80":{"cpu":"0.0","mem":"0.0"},"81":{"cpu":"0.0","mem":"0.0"},"85":{"cpu":"0.0","mem":"0.0"},"90":{"cpu":"0.0","mem":"0.1"},"MiB":{"cpu":"10142.8","mem":"avail"},"PID":{"cpu":"%CPU","mem":"%MEM"},"Tasks:":{"cpu":"stopped,","mem":"0"},"top":{"cpu":"users,","mem":"load"}},"detection_fps":0.0,"detectors":{"cpu":{"detection_start":0.0,"inference_speed":385.17,"pid":220}},"gpu_usages":{"intel-vaapi":{"gpu":"6.17 %","mem":"- %"}},"service":{"last_updated":1704041697,"latest_version":"0.12.1","storage":{"/dev/shm":{"free":2143.7,"mount_type":"tmpfs","total":2147.5,"used":3.8},"/media/frigate/clips":{"free":653426.9,"mount_type":"ext4","total":981813.3,"used":278437.5},"/media/frigate/recordings":{"free":653426.9,"mount_type":"ext4","total":981813.3,"used":278437.5},"/tmp/cache":{"free":1992.4,"mount_type":"tmpfs","total":2000.0,"used":7.6}},"temperatures":{},"uptime":1109,"version":"0.12.1-367d724"}}
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

reolink

**Comment by NickM-27** [comment]
Moved to discussion as it is a question. 

https://deploy-preview-6262--frigate-docs.netlify.app/plus/ Contains a lot of information on this. The model included in frigate is a demo model trained on images from the COCO dataset which means it is not trained on security camera images.

**Reply by mabed-fr** [reply]
Great!

Lot of information !

If I understand correctly with frigate we will move on to a model designed from the start with samples from frigate+ users and many other samples. And with our credit we could orient our models with our detections, is that right? Will non-frigate+ users stay on the model not designed from the beginning?

**Reply by NickM-27** [reply]
to be clear I don't work on Frigate+ and I don't know the full answer, but from what I have seen the default model in Frigate will not be a Frigate+ model but can change if a better freely available and distributable model is identified.

**Reply by mabed-fr** [reply]
Thank you for the lots of information, it’s already very good
